packages/tests/src/helpers/assertions.ts

```diff
@@ -0,0 +1,22 @@
+import ScreenshotService from './../services/ScreenshotService'
+
+// TODO - .env
+const SPECTRE_UI_URL='https://spectre.service.bcdev'
```

Comment:
Is your lint working? I would expect this to fail because it expects whitespaces before and after the `=`, also a missing `;`

Reply:
It did come up with several errors, currently it's clean.

Reply:
Fixed the linting issue, but don't know why the linter didn't pick it up.